cookbook/specs/near.json

```diff
@@ -492,6 +492,19 @@
                             },
                             {
                                 "name": "pruning",
+                                "parse_directive": {
```

Comment:
we have a really easy solution for that.
example from evmos: https://github.com/lavanet/lava/blob/0a14d3465e9e1440a5717ea43f801f7f6aed4506/cookbook/specs/evmos.json#L1560

by defining only the "function_tag": "GET_BLOCK_BY_NUM" you dont need to redefine the whole parse directive.

Reply:
Hi @Yaroms, i i edted like this on https://github.com/lavanet/lava/pull/1498/commits/3dfe7aff5f9dca8c046388c3ededde83e06447a9

Reply:
Hey, i spoke with @omerlavanet, we don't have the code supporting what i said, can you revert the changes back to your previous solution?